bootstore/src/schemes/v0/share_pkg.rs

```diff
@@ -129,7 +129,7 @@ pub fn create_pkgs(
     let shares = rack_secret.split(threshold, total_shares)?;
     let share_digests = share_digests(&shares);
     let mut salt = [0u8; 32];
-    OsRng.fill_bytes(&mut salt);
+    OsRng.try_fill_bytes(&mut salt).expect("fetched random bytes");
```

Comment:
This is fine, but I'm curious why you changed it.

Reply:
Ok, and you wanted to make 0.8 work the same as 0.9 in the codebase. And the failure mode seems better. Thanks!